README.md

```diff
@@ -55,3 +55,13 @@ If it is the first time of deployment, there are 2 ways to upload the package.
   It is because you can not select a project that does not exist in `pypi` yet.
 
 After the first deployment, a new token that has access to the project can be created in (`pypi.org`)[https://pypi.org/] and configured in the repository.
+
+### Branch Protection Rules
+Go to `Settings > Branches` and in the `Branch protection rules` add rule for `main` branch to project it.
+Under `Protect matching branches` setting, select
+- [ ] `Require a pull request before merging`
+- [ ] `Require approvals`
+- [ ] `Require status checks to pass before merging`
+
+      You can either use all checks, or select `required` checks only.
+- [ ] `Require branches to be up to date before merging`
```

Comment:
I don't think we are using the final item (in all projects).

Should we note that one can select the checks only after CI has run for the first time?

Reply:
At least I think we are frequently ignoring it, when PRs are "independent". But it is probably better to enable and follow the setting as you suggest.

Reply:
Yes, I think so too. It'll make it easier to resolve conflicts in advance. I added that line again...!